cmd/juju/subnet/add_test.go

```diff
@@ -237,15 +237,3 @@ func (s *AddSuite) TestRunWithAmbiguousCIDRDisplaysError(c *gc.C) {
 		s.Strings("zone1", "zone2"),
 	)
 }
-
-func (s *AddSuite) TestRunAPIConnectFails(c *gc.C) {
```

Comment:
as explained on IRC, commands should not fail to connect to the API server because an api field in the struct is nil.. because now we have an apiContext struct embedded in the command which is set by the wrapped.. so if we wrap the command we should be able to connect to the API server.